### ISSUE TITLE
Review from Murray Kucherawy (ART)

### ISSUE THREAD

**Issue by beurdouche** [mlswg/mls-architecture#185]
```
Murray Kucherawy has entered the following ballot position for
draft-ietf-mls-architecture-10: No Objection

When responding, please keep the subject line intact and reply to all
email addresses included in the To and CC lines. (Feel free to cut this
introductory paragraph, however.)


Please refer to https://www.ietf.org/about/groups/iesg/statements/handling-ballot-positions/ 
for more information about how to handle DISCUSS and COMMENT positions.


The document, along with other ballot positions, can be found here:
https://datatracker.ietf.org/doc/draft-ietf-mls-architecture/



----------------------------------------------------------------------
COMMENT:
----------------------------------------------------------------------

Thanks to Valery Smyslov for his two ARTART reviews.  I would encourage the
authors of this document to respond to the second one.

This is really well done and easy to read.  Nice work.  I have just a few
things to raise for your consideration beyond what others have said already.

The Abstract says:

"This document describes a general secure group messaging infrastructure and
its security goals."

...but it uses a number of terms that are defined in the MLS protocol document
(Proposal, Commit, etc.).  That means this document isn't as generic as this
text suggests.  This is not a blocker to publication -- the work is clearly
very thorough -- but this sentence sets the wrong expectations, I think.  It
really is more of a reader's guide or a companion specifically to the MLS
protocol document.

Section 2, and in particular 2.1, defines "clients", "groups", and "members",
but in a few spots it seems like they get crossed.  For instance, in Section 2:

* add one or more clients to an existing group;

That should be "members", not "clients", I think.

In Section 4.1:

"When a client wishes to establish a group or add clients to a group, ..."

Isn't it members that are part of groups, not clients?
```

**Comment by rohan-wire** [comment]
Merged PR #188 should address the first comment.

Regarding: clients, groups, and members:
The current wording is correct. You add clients to a group. Once they have been added then they are members (and still clients) but not before.

Ready to close as resolved.

**Comment by beurdouche** [comment]
Verified as resolved ! Thank you !
